src/pages/create-wiki/index.tsx

```diff
@@ -336,17 +336,22 @@ const CreateWikiContent = () => {
     val: string | undefined,
     isInitSet?: boolean,
   ) => {
+    const newVal = val
```

Comment:
```suggestion
    const filteredContent = val
```

renaming so it makes it easy to read what it is doing